rim/tests.py

```diff
@@ -1,3 +1,39 @@
-from django.test import TestCase
+from django.test import TestCase, Client
 
 # Create your tests here.
+class TestHttpResponse(TestCase):
+    # Url dictionary to hold names for existing rim web pages, add as neccessary 
+    urlDictionary = {
+        "Admin Panel": '/admin/',
+        "Home": '/',
+        "Add": '/add/',
+        "Edit": '/edit/1',
+        "Client": '/client/1'
+    }
+
```

Comment:
We should probably have a `setUp()` method that sets up the database with a basic initial state (1 user, 1 equipment, 1 client, etc). And also [logs in](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#django.test.Client.login) the default `self.client` with that one user's credentials.

[Here's just a quick example of what a small `setUp()` method looks like.](https://docs.djangoproject.com/en/3.1/topics/testing/overview/#writing-tests)